docs/pages/eas-update/getting-started.mdx

```diff
@@ -119,7 +119,7 @@ To publish an update to the build, run the following command:
   cmdCopy={null}
 />
 
-Once the update is built and uploaded to EAS and the command completes, force close and reopen your app up to two times to download and view the update.
+Once the update is built and uploaded to EAS and the command completes, force close and reopen your app up to two times to download and view the update. If your app is not updating as expected, validate your configuration [here](/eas-update/debug).
```

Comment:
```suggestion
Once the update is built and uploaded to EAS and the command completes, force close and reopen your app up to two times to download and view the update. If your app is not updating as expected, [validate your configuration](/eas-update/debug).
```

A small suggestion about linking to other docs as per our writing styleguide is to avoid using words like here: https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#linking-to-other-docs